csrc/ops.cu

```diff
@@ -366,10 +361,33 @@ template int get_leading_dim<ROW>(int dim1, int dim2);
 template int get_leading_dim<COL>(int dim1, int dim2);
 template int get_leading_dim<COL32>(int dim1, int dim2);
 
+// TODO: Check overhead. Maybe not worth it; just check in Python lib once,
+//       and avoid calling lib functions w/o support for them.
+// TODO: Address GTX 1660, any other 7.5 devices maybe not supported.
+inline bool igemmlt_supported() {
+  int device;
+  int ccMajor;
+
+  CUDA_CHECK_RETURN(cudaGetDevice(&device));
+  CUDA_CHECK_RETURN(cudaDeviceGetAttribute(&ccMajor, cudaDevAttrComputeCapabilityMajor, device));
+
+  if (ccMajor >= 8)
+    return true;
+
+  if (ccMajor < 7)
+    return false;
+
+  int ccMinor;
+  CUDA_CHECK_RETURN(cudaDeviceGetAttribute(&ccMinor, cudaDevAttrComputeCapabilityMinor, device));
+
+  return ccMinor >= 5;
+}
```

Comment:
Using this as more of a sanity check right now. I'd expect we wouldn't be calling `transform`, `spmm_coo`, or `igemmlt` with devices that don't support it, but I haven't verified this. In particular the `spmm_coo` function is one that I am not so sure about.